### PR TITLE
ERCOT: get_load uses todays-outlook.json

### PR DESCRIPTION
**Pull request by jcsalterego**
Addresses #99 which pulls from https://www.ercot.com/api/1/services/read/dashboards/todays-outlook.json for today and latest.

The numbers differ (less than 3%) from the old endpoint, but I confirmed it matches the Demand trend on the associated dashboard page - https://www.ercot.com/gridmktinfo/dashboards/supplyanddemand:

```python
iso = gridstatus.Ercot()
today = iso._get_load_json("currentDay")
today2 = iso._get_todays_outlook_non_forecast("today").rename(columns={"demand": "Load"})
df = pd.merge(today, today2, suffixes=("_control", "_experiment"), on="Time")
df["Load_delta"] = df["Load_control"] - df["Load_experiment"]
df["Load_delta_pct"] = df["Load_delta"] / df["Load_control"] * 100.0
df = df[["Time", "Load_control", "Load_experiment", "Load_delta", "Load_delta_pct"]]
print(df.sort_values(by="Time").reset_index(drop=True).to_string())
```
```
                        Time  Load_control  Load_experiment  Load_delta  Load_delta_pct
0  2022-12-27 01:00:00-06:00      46493.58            46447       46.58        0.100186
1  2022-12-27 02:00:00-06:00      46660.27            47015     -354.73       -0.760240
2  2022-12-27 03:00:00-06:00      47486.36            47957     -470.64       -0.991106
3  2022-12-27 04:00:00-06:00      48385.85            48816     -430.15       -0.889000
4  2022-12-27 05:00:00-06:00      50027.51            51025     -997.49       -1.993883
5  2022-12-27 06:00:00-06:00      52589.68            53980    -1390.32       -2.643713
6  2022-12-27 07:00:00-06:00      55716.44            57158    -1441.56       -2.587315
7  2022-12-27 08:00:00-06:00      57876.19            58455     -578.81       -1.000083
8  2022-12-27 09:00:00-06:00      57562.39            56452     1110.39        1.929020
9  2022-12-27 10:00:00-06:00      55368.85            53768     1600.85        2.891247
10 2022-12-27 11:00:00-06:00      52569.46            51029     1540.46        2.930333
11 2022-12-27 12:00:00-06:00      49921.23            48605     1316.23        2.636614
12 2022-12-27 13:00:00-06:00      47462.76            46323     1139.76        2.401377
13 2022-12-27 14:00:00-06:00      45519.76            44658      861.76        1.893156
14 2022-12-27 15:00:00-06:00      44264.55            43744      520.55        1.175997
```

### CODE REVIEW

**Review by kmax12 — APPROVED** [review]
Looks great!

File: gridstatus/tests/test_ercot.py [hunk 1]
```diff
@@ -74,6 +74,18 @@ def test_ercot_get_load_today():
     assert df["Time"].unique()[0].date() == today
 
 
+def test_ercot_get_load_latest():
+    expected_keys = {
+        "Time",
+        "Load",
```
Comment (kmax12) [inline gridstatus/tests/test_ercot.py:80]: this is a weirdness in the current gridstatus api, but `get_load("latest")` returns a dictionary with all lower case keys for all the ISOs. 

this is why `test_get_latest_load` in `test_isos.py` is failing after this change

**Review by kmax12 — CHANGES_REQUESTED** [review]

File: gridstatus/tests/test_ercot.py (hunk 1 above)
Comment (jcsalterego) [reply gridstatus/tests/test_ercot.py:80]: Ahh my bad.

Comment (jcsalterego) [reply gridstatus/tests/test_ercot.py]: Fixed this. I misread the test failure as an unrelated issue.

However, this latest failure is unrelated, I believe: https://github.com/kmax12/gridstatus/actions/runs/3790437409/jobs/6445062172

Comment (kmax12) [reply gridstatus/tests/test_ercot.py]: ya, i agree the latest failure all look unrelated. tests seem to really be running slow today 😢

**Review by kmax12 — APPROVED** [review]

**Pull request merged**